guides/source/getting_started.md

```diff
@@ -2120,6 +2120,10 @@ Authentication challenge:
 
 ![Basic HTTP Authentication Challenge](images/getting_started/challenge.png)
 
+NOTE: When using HTTP Basic Authentication, the authentication popup will appear
+only once for a user and will not appear for further requests to authenticated
+controller actions, unless the user is signed out.
```

Comment:
I feel like the reader might be left with the question: "But how does a user sign out?"  Also, instead of making this a `NOTE`, what about making it the conclusion of the narrative:

```suggestion
After entering the correct user name and password, you will remain authenticated
until a different user name and password is required or the browser is closed.
```